src/Pill/Pill.tsx

```diff
@@ -12,13 +12,11 @@ export interface PillProps
     ChipProps,
     | "onDelete"
     | "avatar"
-    | "icon"
     | "disabled"
     | "deleteIcon"
     | "color"
     | "disabled"
     | "classes"
-    | "clickable"
```

Comment:
Clickable is not needed here. Clickable css class is applied by passing onClick parameter

Reply:
How about adding explicit class with `cursor: inherit;`?